### PR TITLE
fix: ory summit blogpost

### PR DESCRIPTION
**Pull request by vinckr**
- fix: add ory summit blogpost
- fix: add ory summit blogpost images

@reviewers: Please take a look if everything is formatted correctly, the pictures are OK and there is nothing major I left out.

### CODE REVIEW

File: src/markdown/blog/digital-developer-conference-guide-tips.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+Normally such events are executed with three times the amount of resources and time, so we had to find an efficient framework for making our first conference a success. In retrospect, we learned loads about how to run a live digital event for developer communities, and that’s what we want to share in this article.
+
+### Why we held the Summit
+
+Digital human communities usually center around recurring events that bring them together in one place to share a communal spirit, exchange ideas, and get to know each other. The Ory Summit is the first recurring event for the Ory Developer Community. 
+
+Ory services provide free and open identity infrastructure for a lot of different platforms – examples include an Anime media aggregator called [Animeshon](https://www.ory.sh/summit/2021/google-iam-from-scratch/), or [SumUp](https://www.ory.sh/summit/2021/self-service-oidc-terraform-hydra/), a payments company specializing in solutions for small businesses. We thought our developer community would enjoy more opportunities to share the challenges and solutions that arise as they integrate Ory services. 
+
+### So you want to bootstrap a developer conference?
```
Comment (jfcurran) [inline src/markdown/blog/digital-developer-conference-guide-tips.md:36]: List Items for 1. The Basics are formatted the following way

- **Know your audience** It is hard to overstate how important this is. 

List items for 2. The Stack and 3. The preparation are formatted as follows

- Chat  
We use Slack for chats.

I suggest to got with a formatting that emphasises the main point of the list item

- **Know your audience**  
 It is hard to overstate how important this is. 
 
when writing markdown for the blog, a double space behind a line will render it as intened `  ` 

- **Know your audience**  
 It is hard to overstate how important this is. 
vs
- **Know your audience** It is hard to overstate how important this is. 

this doesn't apply to every instance of list items, but I appreciate consistent formatting when digesting information of equal importance

File: src/markdown/blog/digital-developer-conference-guide-tips.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+- Code of Conduct & Inclusivity
+- Audio & Video Quality Optimization
+- Visual Branding
+- Landing Page
+- Interaction with Audience/Q&A
+- Interactive Workshops
+- Merchandise
+- Sponsors
+- Tickets (Free/Paid)
+...and much more
```
Comment (jfcurran) [inline src/markdown/blog/digital-developer-conference-guide-tips.md:106]: this liste item isn't rendered as intended and is appended to - Tickets (Free/Paid)

File: src/markdown/blog/digital-developer-conference-guide-tips.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+publishedAt: '2022-01-19'
+author: vinckr
+
+overline: |
+  Community meetup tutorial
+---
+
+We were stoked to pull off the first Ory Summit 2021 in October, bringing together an all-star group of developers who presented on the diverse ways in which they use Ory open source products. 
+We bootstrapped the two-day Ory Summit with a core team of three people, supported by the rest of Ory, plus the presenters, members of our open source community who were generous with their time and energy. 
+
```
Comment (jfcurran) [inline src/markdown/blog/digital-developer-conference-guide-tips.md:25]: I believe adding an image here might make the ory summit more tangible within the article. the image we used as a splash screen would work as an example

File: src/markdown/blog/digital-developer-conference-guide-tips.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  4. More content for non-technical people. 
+  5. More workshops & beginner sessions.
+
+- Analyze the data:  
+This depends on what you collected. Our main source of data was the exit interview and some attendance metrics collected on the platform.
+
+- Organize the next event:  
+Take a deep breath, and begin planning your next event. 
+
+### Ory Summit 2022 ⛰️
```
Comment (jfcurran) [inline src/markdown/blog/digital-developer-conference-guide-tips.md:132]: nice emoji 👀 
maybe it is sensible to consistently add emojis or omit them entirely. I think it adds to the article, but I personally dont like emojis in serious articles - this discussion is not in scope for this pull request, but it is something we need to formalize going forward

File: src/markdown/blog/digital-developer-conference-guide-tips.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+In light of how much fun we had at our first event, we are stoked for the Ory Summit 2022. Preparation has already begun and we will soon publish a call for papers and more. If the global health situation permits, there will be an option to attend the Ory Summit in person this year. It will take place in Q2/Q3 – the exact date coming soon. 
+
+We have many things planned for this year. While not wanting to spoil the surprise, I think this is going to be the biggest Ory event ever 😁.
```
Comment (jfcurran) [inline src/markdown/blog/digital-developer-conference-guide-tips.md:136]: 🙂

File: src/markdown/blog/digital-developer-conference-guide-tips.md (hunk 4 above)
Comment (vinckr) [reply src/markdown/blog/digital-developer-conference-guide-tips.md:132]: I like emojis and I think they are important for this kind of content 😉 
Our blog is very dry as it is IMO.

**Comment by vinckr** [comment]
Thanks for your review @jfcurran, I made some changes.

**Review by jfcurran — APPROVED** [review]
😎 so good - makes me remember how much of a blast the summit was. Thank you for this nice article! i think lots of people will be happy reading this prior to bootstrapping their own event

**Review by jfcurran — APPROVED** [review]
LGTM 👍

**Review by zeropaper — APPROVED** [review]
LGTM :+1:

**Pull request merged**